### DISCUSSION TITLE
Sign in with phone number and password doesn't work

### DISCUSSION THREAD

**Discussion by nickbyte** [supabase/supabase#3252] [Questions] [answered]
After creating an user using phone number and password, using

`supabase.auth.signUp({phone, password})`

Getting an error saying 'invalid email/password' error when i sign in

`supabase.auth.signIn({phone, password})`

Any one has been able to setup phone + password login?

**Comment by dshukertjr** [comment, marked as answer]
Hi @nickbyte!

Could you share your code where you are trying to sign user up with phone number and password? Could your `phone` variable be undefined by chance?

**Reply by nickbyte** [reply]
Resolved, Thank you

**Reply by dshukertjr** [reply]
@nickbyte Glad to hear that! What was just for reference, what was the cause? Was it because it was before verifying OTP maybe?
